cmd/podman/common/volumes.go

```diff
@@ -104,12 +104,12 @@ func parseVolumes(volumeFlag, mountFlag, tmpfsFlag []string, addReadOnlyTmpfs bo
 	// Check for conflicts between named volumes and mounts
```

Comment:
We need conflict checks for the new overlay volumes, as we have for named volumes and normal bind mounts right now.